client/src/app/components/gui-editor/gui-editor.events.ts

```diff
@@ -0,0 +1,11 @@
+import {SidebarType} from "./sidebar/editor-sidebar.component";
+
+export enum GuiEditorEventType {
+    showSidebar
```

Comment:
Convention for enum values is [PascalCase](https://www.typescriptlang.org/docs/handbook/enums.html), I believe.